docs/08-developer-guide/04-project/03-release-guide.md

```diff
@@ -4,6 +4,372 @@ sidebar_label: Release Manager Guide
 
 # Apache Release Manager Guide
 
-**TODO:** File a subtask under [HDDS-9861](https://issues.apache.org/jira/browse/HDDS-9861) and complete this page or section.
+This document describes the process to release Apache Ozone. The process is not yet scripted, and the documentation is **a work in progress**
 
-Port from confluence
+## Pre-Requisite
```

Comment:
I think we should combine this part under `Publish Your GPG Key` since it is just one line, and then move `Publish Your GPG Key` under `Pre-Vote`. IMO this makes the mini-map flow better.